python/cuml/test/test_pickle.py

```diff
@@ -443,3 +443,26 @@ def create_mod():
     def assert_model(pickled_model, data):
         compare_svm(result["model"], pickled_model, data[0], data[1], cmp_sv=0,
                     dcoef_tol=0)
+
+    pickle_save_load(tmpdir, create_mod, assert_model)
+
+
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('nrows', [unit_param(500)])
+@pytest.mark.parametrize('ncols', [unit_param(16)])
+@pytest.mark.parametrize('n_info', [unit_param(7)])
+def test_svr_pickle(tmpdir, datatype, nrows, ncols, n_info):
```

Comment:
Instead of this we could make add `SVR` to `regression_models`, the only problem there is the nrows=500k stress param for `test_regressor_pickle` would take too long to execute.

Reply:
I think it is fine to leave this test as is for now